randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -143,7 +143,7 @@ Extra - room_id: [3]
       Any of the following:
           Wall Jump (Beginner) or Can Climb High Jump Wall
           Speed Booster and Shinespark Tricks (Beginner)
-          After Level 2 Locks Released and Stand On Frozen Enemies (Beginner)
+          After Level 2 Locks Released and Stand On Frozen Enemies (Beginner) and Can Freeze Enemies
```

Comment:
I assume this wasn't meant for this PR, but oh well

Reply:
It was a part of `test_fusion.py`.